flathunter/proxies.py

```diff
@@ -0,0 +1,18 @@
+""" Gets proxies """
+import requests
+from lxml.html import fromstring
+
+def get_proxies():
+    """
+    Gets random, free proxies
+    """
+    url = 'https://free-proxy-list.net/'
+    response = requests.get(url)
```

Comment:
I love that we get the list of free proxies for our scraping app by scraping a website listing proxies :)